src/PerformanceTests/Tests/Permutations/PermutationParser.cs

```diff
@@ -10,17 +10,24 @@ public static class PermutationParser
 
         public static string ToArgs(Permutation instance)
         {
-            return "--tests:" + string.Join(Seperator, instance.Tests) + " --variables:" + ToString(instance);
+            return "--tests:" + string.Join(Seperator, instance.Tests) + 
+                   " --category:" + instance.Category +
+                   " --description:\"" + instance.Description + "\"" +
+                   " --variables:" + ToString(instance);
```

Comment:
Why a : here, but a = in the other arguments? The parser will handle both, but = seems more consistent?